wire-ios-request-strategy/Sources/Notifications/NotificationStreamSync.swift

```diff
@@ -119,6 +119,9 @@ public class NotificationStreamSync: NSObject, ZMRequestGenerator, ZMSimpleListR
     @objc(shouldParseErrorForResponse:)
     public func shouldParseError(for response: ZMTransportResponse) -> Bool {
         notificationStreamSyncDelegate?.failedFetchingEvents()
+        guard response.apiVersion != APIVersion.v3.rawValue else {
```

Comment:
This change will be for API V3 and above, so we want to fallback to the old implementation only if the API version is < 3
```suggestion
        guard response.apiVersion < APIVersion.v3.rawValue else {
```